tests/ui_tests/test_admin_page.py

```diff
@@ -365,6 +368,41 @@ def test_ldap_user_change_is_forbidden(self, users_page: AdminUsersPage, ldap_us
             ), f'User {ldap_user_in_group["name"]} was not created'
         users_page.check_ldap_user(ldap_user_in_group["name"])
 
+    def test_add_user_to_group(self, user, users_page, sdk_client_fs):
+        """Add group for user"""
+        params = {'name': 'test', 'email': 'test@test.ru'}
+        test_group = sdk_client_fs.group_create('test_group')
+        users_page.update_user_info(
+            user.username,
+            first_name=params['name'],
+            last_name=params['name'],
+            email=params['email'],
+            group=test_group.name,
+        )
+        with allure.step(f'Check user {user.username} is listed in users list with changed params'):
+            user_row = users_page.get_user_row_by_username(user.username)
+            assert test_group.name in user_row.text, "User group didn't changed"
+            assert params['email'] in user_row.text, "User email didn't changed"
+
+    @pytest.mark.ldap()
+    @pytest.mark.usefixtures("configure_adcm_ldap_ad")
+    def test_add_ldap_group_to_users(self, user, users_page, sdk_client_fs, ldap_user_in_group):
+        """Check that user can't add ldap group to usual user"""
+        users_page.header.wait_success_job_amount_from_header(1)
```

Comment:
can you please explain what are we waiting here?

Reply:
add allure step